includes/class-wc-subscriptions-email-notifications.php

```diff
@@ -306,4 +309,53 @@ public static function add_settings( $settings ) {
 		WC_Subscriptions_Admin::insert_setting_after( $settings, WC_Subscriptions_Admin::$option_prefix . '_miscellaneous', $notification_settings, 'multiple_settings', 'sectionend' );
 		return $settings;
 	}
+
+	/**
+	 * Maybe add an admin notice to inform the store manager about the existance of the notifications feature.
+	 */
+	public static function maybe_add_admin_notice() {
+
+		// If the notifications feature is enabled, don't show the notice.
+		if ( self::notifications_globally_enabled() ) {
+			return;
+		}
```

Comment:
Maybe the filter can be hooked conditionally only if notifications are disabled? Not sure what's more WordPress-y way to solve this, but every function call costs a bit of CPU.

Reply:
Honestly, you’re not wrong here. However, because the constructor runs early (plugins_loaded), I’m more inclined to keep it as is, in case anyone wants to filter it during `init`.